sec_certs/dataset/cve.py

```diff
@@ -147,9 +147,35 @@ def from_json(cls, input_path: Union[str, Path]):
             dset = json.load(handle, cls=CustomJSONDecoder)
         return dset
 
-    def get_cve_ids_for_cpe_uri(self, cpe_uri: str) -> Optional[Set[str]]:
+    def _get_cve_ids_for_cpe_uri(self, cpe_uri: str) -> Optional[Set[str]]:
         return self.cpe_to_cve_ids_lookup.get(cpe_uri, None)
```

Comment:
What would be the consequences of returning empty set by default instead of None?